packages/interpretations/src/components/details/Description.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import i18n from '@dhis2/d2-i18n';
+
+export const Description = model => {
```

Comment:
This file returns inconsistent value types. It may return a component `<i>No Description</i>`, but it may also just return a string.

Reply:
I added a span to the other return values so it always returns a JSX element.